modules/custom/az_paragraphs/az_paragraphs_link_group/az_paragraphs_link_group.module

```diff
@@ -5,10 +5,6 @@
  * Contains az_paragraphs_link_group.module.
  */
 
-use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
```

Comment:
This can be  fixed here: https://github.com/az-digital/az_quickstart/issues/3210